charts/seed-controlplane/charts/etcd/values.yaml

```
@@ -53,6 +57,16 @@ etcdResources:
     cpu: 900m
     memory: 3G
 
+sidecarResources:
```

Can we adapt such as? Also applies for `etcdResources`.

```suggestion
sidecar:
  resources:
    limits:
      cpu: "1"
      memory: 10G
    requests: 
      cpu: 23m 
      memory: 128Mi
```

---

done.